.github/workflows/mac-artifact-jfrog.yml

```diff
@@ -0,0 +1,108 @@
+name: Mac Artifact
```

Comment:
Probably this mac artifact should be removed from the pr until the mac is ready to use.